index.js

```diff
@@ -1,17 +1,66 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = items;
+    this.length = length;
+  }
 
-  add(item) {}
+  add(item) {
+    //agregamos elemento al array
+    this.items.push(item);
+    //ordenamos de manera ascendente
+    const sortArray = this.items.sort((a, b) => {
+      return a - b;
+    });
+    //actualizamos la propiedad de la longitud
+    this.length = sortArray.length;
+    return sortArray;
+  }
 
-  get(pos) {}
+  get(pos) {
+    //lanzar error en caso de que no se encuentre el indice
+    if (pos >= this.length || pos < 0) {
```

Comment:
Menos 1 por que length nos da el no de elementos pero los indices empiezan en 0 😉 
```suggestion
    if (pos >= this.length - 1 || pos < 0) {
```